addons/mail/static/src/core/common/thread_icon.xml

```diff
@@ -5,9 +5,14 @@
         <div class="o-mail-ThreadIcon d-flex justify-content-center flex-shrink-0" t-att-class="props.className">
             <t t-set="largeClass" t-value="props.size === 'large' ? 'fa-lg' : ''"/>
             <t t-if="props.thread.channel?.channel_type === 'channel'">
-                <div t-if="props.thread.group_public_id" class="fa fa-fw fa-hashtag" t-att-class="largeClass" t-att-title="props.thread.accessRestrictedToGroupText"/>
+                <div t-if="props.thread.group_public_id" class="fa fa-fw fa-lock" t-att-class="largeClass" t-att-title="props.thread.accessRestrictedToGroupText"/>
                 <div t-else="" class="fa fa-fw fa-globe" t-att-class="largeClass" title="Public Channel"/>
             </t>
+            <t t-elif="props.thread.channel?.channel_type === 'group'">
+                <t name="group">
+                    <div class="o-mail-ThreadIcon oi fa-fw oi-users" t-att-class="largeClass" title="Grouped Chat"/>
+                </t>
+            </t>
```

Comment:
```suggestion
            <div t-elif="props.thread.channel?.channel_type === 'group'" class="o-mail-ThreadIcon oi fa-fw oi-users" t-att-class="largeClass" title="Grouped Chat" t-ref="groupIcon"/>
```

why all the extra `<t/>`?

Reply:
The patch uses `replace`, so the `div` won't be an issue. As for the `t-if`, It's indeed more robust to keep the condition blocks in place. But then, the `<t t-name>` part can still be on the div.